scripts/check-legend-application-documentation-broken-links.js

```diff
@@ -0,0 +1,81 @@
+const fs = require("fs");
+
+const DOC_WEBSITE_URL = "https://legend.finos.org/";
+const APPLICATION_DOC_DIRECTORY =
+  "website/static/resource/studio/documentation/";
+const WEBSITE_CONTENT_DIRECTORY = "./website/build/";
+
+var files = fs.readdirSync(APPLICATION_DOC_DIRECTORY);
+var brokenUrls = [];
+
+for (let i = 0; i < files.length; i++) {
+  if (!files[i].endsWith(".json")) continue;
+  var jsonObject = JSON.parse(
+    fs.readFileSync(APPLICATION_DOC_DIRECTORY + files[i], "utf8")
+  );
+
+  const keys = Object.keys(jsonObject.entries);
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
+    if (jsonObject.entries[key].url == undefined) {
+      continue;
+    }
+    lookForUrl(jsonObject.entries[key].url);
+  }
+}
+
+if (brokenUrls.length !== 0) {
+  throw new Error("Broken link(s) found: " + brokenUrls);
```

Comment:
Here, you can print it nicely, something like

```suggestion
  throw new Error(`Found ${brokenUrls.length} broken link(s):\n${brokenUrls.join('\n')}`);
```